budgetportal/tests/test_pages.py

```diff
@@ -61,30 +60,125 @@ def test_overview_page(self):
         self.assertEqual(content['financial_years'][-1]['id'], '2019-20')
         self.assertEqual(content['financial_years'][0]['id'], '2016-17')
 
-
-    def test_department_list_page(self):
+    def test_department_detail_page_yaml(self):
         """Test that it exists and that the correct years are linked"""
         c = Client()
-        response = c.get('/2019-20/departments.yaml')
+        response = c.get('/2019-20/national/departments/the-presidency.yaml')
         content = yaml.load(response.content)
         self.assertEqual(content['financial_years'][-1]['id'], '2019-20')
         self.assertEqual(content['financial_years'][0]['id'], '2016-17')
 
-    def test_department_list_page_not_latest(self):
-        """Test that it exists and that the correct years are linked"""
+    def test_department_detail_page(self):
+        """Test that it loads and that some text is present"""
         c = Client()
-        response = c.get('/2015-16/departments.yaml')
-        content = yaml.load(response.content)
-        self.assertEqual(content['financial_years'][-1]['id'], '2019-20')
-        self.assertEqual(content['financial_years'][0]['id'], '2016-17')
+        response = c.get('/2019-20/national/departments/the-presidency')
+        content = response.content
+        self.assertTrue(content.find('The Presidency budget data for the 2019-20 financial year'))
 
-    def test_department_detail_page(self):
-        """Test that it exists and that the correct years are linked"""
+    def test_about_page(self):
+        """Test that it loads and that some text is present"""
         c = Client()
-        response = c.get('/2019-20/national/departments/the-presidency.yaml')
-        content = yaml.load(response.content)
-        self.assertEqual(content['financial_years'][-1]['id'], '2019-20')
-        self.assertEqual(content['financial_years'][0]['id'], '2016-17')
+        response = c.get('/about')
+        content = response.content
+        self.assertTrue(content.find('Learn more about the new Online Budget Data Portal'))
```

Comment:
no need to change this now, but heads-up in future you might wanna use self.assertContains for SimpleTestCase because it does the last two lines for you and a bunch of other stuff too like better failure feedback https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains